nwb_healthstatus/__main__.py

```diff
@@ -16,7 +16,7 @@ def sample():
 
 @sample.command()
 #@click.option("-e", "--environment")
-@click.option("--overwrite", is_flag=True)
+@click.option("--overwrite", is_flag=True, default=True)
```

Comment:
```suggestion
@click.option("--overwrite", is_flag=True)
```
nope -- by default we do not want to overwrite the files.  Should be explicitly requested with this flag